cli/package.json

```diff
@@ -38,7 +38,7 @@
     "commander": "^8.1.0"
   },
   "devDependencies": {
-    "@phated/pkg": "5.9.1-grain",
+    "@spotandjake/pkg": "6.4.0",
```

Comment:
Could we instead fork this in the Grain org?

Reply:
Below should be a working workflow to fork it into the grain org (obviously using the pkg-fetch versions that match with upstream):
1) Fork pkg-fetch
2) Run build_all with no release
3) Update shas
4) make commit
5) Publish the release
6) publish to npm with npm run publish, publish as v3.6 not v3.6.0 in the tag

----

7) Fork pkg
8) Switch to use @grain/pkg-fetch
9) Regen yarn config
10) Test
11) Publish

From what I remember we need a `github PAT` and `npm PAT` for the release process. I'm happy to fork it over and everything but I don't think I currently have permissions todo that.

Reply:
This should be ready to go now.